lib/windows/launcher/actions/desktopShortcutActions.js

```diff
@@ -64,6 +65,12 @@ function createAppShortcut(name) {
     };
 }
 
+/**
+ * Create desktop shortcut on Windows
+ *
+ * @param {app} app, create desktop shortcut for which app.
+ * @returns {dispatch} dispatch action in Redux.
```

Comment:
The text inside braces in JSDoc should be types. In this case `@param {Object} app` and `@returns {Function}`.

Reply:
Fixed